go/README.md

```diff
@@ -1 +1,177 @@
+		db.Initialize() could be db.Init()
+- no getters (the fact it begins with a capital and returns value, makes it a getter)
+- setters can be used
+- if crud, follow crud order inside source? (thoughts about this?)
+- global vs local
+  - first all global functions and methods (they start with capitals)
+  - local function below that (they start with lowercase)
+- return var's:
+  - when returning an error, it is always the last value that returns
+
+## Testing
+
+- code coverage (100% of relevant)
+- no . imports
+
+- Handlers should only test the basic working.
+- Services should test every scenario possible and should be tested with 100% codecoverage.
+  - if there are model validations, these should be tested too.
+- Repositories should test every scenario possible and should be tested with 100% codecoverage.
+- 
+
+## Comments
+
+- package
+  - doc
+    - Describes why the package exists
+    - it has the package format:
+      - Package the_name the_rest_of_the_text.
+- section
+  - special format:
+	/* ------------------------------- Imports --------------------------- */
+	/* ------------------------------- Constants/Types/Variables --------------------------- */
+	/* ------------------------------- Methods/Functions --------------------------- */
+- variable
+  - single line
+    - format of that:
+    // explain what it does and/or why it exists
+
+- function/method
+  - multi line
+    - if the function/method is public:
+      - first line is a complete sentence, starting with capital, ending with dot.
+    - if it is private:
+      - first line is a complete sentence, starting with lowercase, not ending with dot.
+    - is written in third-person singular with minimal 4 words
+     - format of that
+	/*
+	YourFunction does something spectacular.
+	*/
+
+## Variable naming
+
+A variable inside a function can have a short name like "a", if it is clear what it is used for.
+Like in this example:
+a := asserts.New(t)
+a.Equals("Check", someVar)
+
+or in a loop, like in this example:
+for k,v := range records {
+	println("key:", k)
+	println("val:", v)
+}
+
+In any other case, it is wise to name the variable after the thing it contains.
+If is can hold a collection of things, name it plural, like "users".
+If it can only hold one value, name it singular, like "user".
+if it is a boolean var, start it with something like "has", or "is".
+In some checks, you will find a boolean var like "ok",
+which is mostly used for checking if an element is present in a map.
+
+It's also common use, when a variable name holds an abbreviation, that the letters are all capitals, or all lowercase.
+like "SQLStatement" or "sqlStatement"
+
+## linting
+
+It is nice to start a linter now and then.
+make check, or make check2 will run the linters and will check your files for "mistakes".
+Try to improve your code with the hints you get from them.
+
+## Code Quality
+
+I wrote a lint tool, which also checks for cognitive complexity.
```

Comment:
I guess sonarQube will be sufficient

Reply:
I want to know how the sonarcube linting can be changed. (is the change of the configuration a project setting that can be exported/imported (and can be checked in with the project itself?)